libs/core/src/lib/file-uploader/file-uploader.component.html

```diff
@@ -0,0 +1,52 @@
+<div
+    style="display: block;"
```

Comment:
why style being provided inline rather than in css?

Reply:
removed

Reply:
i still see it inline